File: src/main/java/olivermakesco/de/refmagic/effects/SmeltingAura.java
package olivermakesco.de.refmagic.effects;

import net.minecraft.entity.effect.StatusEffect;
import net.minecraft.entity.effect.StatusEffectType;

public class SmeltingAura extends StatusEffect {
    public SmeltingAura() {
        super(StatusEffectType.BENEFICIAL, 0xFF4400);
    }

    @Override
    public String getTranslationKey() {
        return "refmagic.effects.smelting";
    }
}

File: src/main/kotlin/olivermakesco/de/refmagic/effects/SmeltingAura.kt
package olivermakesco.de.refmagic.effects

import net.minecraft.entity.effect.StatusEffect
import net.minecraft.entity.effect.StatusEffectType

class SmeltingAura : StatusEffect(StatusEffectType.BENEFICIAL, 0xFF4400) {
    override fun getTranslationKey(): String {
        return "refmagic.effects.smelting"
    }
}